{
	"id": "oval:org.almalinux.alsa:def:20244260",
	"version": "636",
	"class": "patch",
	"metadata": {
		"Title": "ALSA-2024:4260: python-idna security update (Moderate)",
		"reference": [
			{
				"ref_id": "ALSA-2024:4260",
				"source": "ALSA",
				"ref_url": "https://errata.almalinux.org/8/ALSA-2024-4260.html"
			},
			{
				"ref_id": "CVE-2024-3651",
				"source": "CVE",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2024-3651"
			},
			{
				"ref_id": "ALSA-2024:4260",
				"source": "ALSA",
				"ref_url": "https://errata.almalinux.org/8/ALSA-2024-4260.html"
			}
		],
		"Description": "The hsakmt packages include a thunk library for AMD's Heterogeneous System Architecture (HSA) Linux kernel driver (amdkfd).\n\nSecurity Fix(es):\n\n* python-idna: potential DoS via resource consumption via specially crafted inputs to idna.encode() (CVE-2024-3651)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
		"advisory": {
			"from": "packager@almalinux.org",
			"Severity": "Moderate",
			"Rights": "Copyright 2024 AlmaLinux OS",
			"issued": {
				"date": "2024-07-02"
			},
			"updated": {
				"date": "2024-07-02"
			},
			"affected_cpe_list": {
				"Cpe": [
					"cpe:/a:almalinux:almalinux:8",
					"cpe:/a:almalinux:almalinux:8::appstream",
					"cpe:/a:almalinux:almalinux:8::powertools",
					"cpe:/a:almalinux:almalinux:8::highavailability",
					"cpe:/a:almalinux:almalinux:8::nfv",
					"cpe:/a:almalinux:almalinux:8::realtime",
					"cpe:/a:almalinux:almalinux:8::resilientstorage",
					"cpe:/a:almalinux:almalinux:8::sap",
					"cpe:/a:almalinux:almalinux:8::sap_hana",
					"cpe:/a:almalinux:almalinux:8::supplementary",
					"cpe:/o:almalinux:almalinux:8",
					"cpe:/o:almalinux:almalinux:8::baseos"
				]
			},
			"bugzilla": [
				{
					"href": "https://bugzilla.redhat.com/2274779",
					"id": "2274779"
				}
			],
			"cve": [
				{
					"text": "CVE-2024-3651",
					"public": "20240412",
					"href": "https://access.redhat.com/security/cve/CVE-2024-3651",
					"impact": "Moderate",
					"cwe": "CWE-400",
					"cvss3": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
				}
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterions": [
							{
								"test_ref": "oval:org.almalinux.alsa:tst:2020034810110",
								"comment": "python3-idna is signed with AlmaLinux OS 8 key (51d6647ec21ad6ea)"
							},
							{
								"test_ref": "oval:org.almalinux.alsa:tst:2020034810111",
								"comment": "python3-idna is signed with AlmaLinux OS 8 key (2ae81e8aced7258b)"
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:org.almalinux.alsa:tst:20244260001",
						"comment": "python3-idna is earlier than 0:2.5-7.el8_10"
					},
					{
						"test_ref": "oval:org.almalinux.alba:tst:20191992003",
						"comment": "AlmaLinux 8 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.almalinux.alba:tst:20191992005",
				"comment": "AlmaLinux must be installed"
			}
		]
	}
}
